docs/source/api/v4/user_current.rst

```diff
@@ -44,7 +44,7 @@ Response Structure
 :fullName:          The user's full name, e.g. "John Quincy Adams"
 :gid:               A deprecated field only kept for legacy compatibility reasons that used to contain the UNIX group ID of the user
 :id:                An integral, unique identifier for this user
-:lastAuthenticated: The date and time at which the user was last authenticated, in :ref:`RFC3339`
+:lastAuthenticated: The date and time at which the user was last authenticated, in :rfc:`RFC3339`
```

Comment:
> `invalid RFC number RFC3339`

You don't need to put "RFC" in the argument to `:rfc`, that's implicit. The argument is just the number, like:
```rst
:rfc:`3339`
```